core-library/src/main/java/org/silverpeas/core/index/indexing/model/DidYouMeanIndexer.java

```diff
@@ -76,7 +76,7 @@ public static void createSpellIndex(String field, String originalIndexDirectory,
       return;
     }
     // initializes local variable
-    IndexReader indexReader = null;
+    /*IndexReader indexReader = null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3ACommentedOutCodeLine)